.docker/aiida-core-base/Dockerfile

```
@@ -137,7 +137,7 @@ RUN set -x && \
         mamba && \
     rm micromamba && \
     # Pin major.minor version of python
-    mamba list python | grep '^python ' | tr -s ' ' | cut -d ' ' -f 1,2 >> "${CONDA_DIR}/conda-meta/pinned" && \
+    mamba list python | grep -oP 'python\s+\K[\d.]+' | tr -s ' ' | cut -d ' ' -f 1,2 >> "${CONDA_DIR}/conda-meta/pinned" && \
```

This is the text from new `mamba list`

```
List of packages in environment: "/opt/conda"

  Name    Version  Build               Channel    
────────────────────────────────────────────────────
  python  3.10.13  hd12c33a_1_cpython  conda-forge
```

It added spaces indentation in front.

---

I was think about to add a comment with the text above. But I think it makes Dockerfile more confusing. 
The problem is the spaces indentation in front but the line is already indented. 
I'd rather not add these comment.

---

Okay can you add this to the commit message at least?

---

I think using awk for this (which ignores whitespace by default) would be much easier

```suggestion
    mamba list python | awk '$1 == "python" {print $2}' >> "${CONDA_DIR}/conda-meta/pinned" && \
```

---

Sorry @danielhollas, I just saw your comment. If we remember next time when changing Dockerfile, will use `awk`. I did not familiar with awk :cry: